gadgets/trace_tcpretrans.yaml

```diff
@@ -0,0 +1,33 @@
+name: tcpretrans
+description: trace TCP retransmissions
+columns:
+  - name: timestamp
+    template: timestamp
+    order: 900
```

Comment:
I had to comment this out to make it work:

```bash
INFO[0000] Experimental features enabled                
Error: calling custom parser: adding fields: duplicate column name "timestamp"
exit status 1
```

I'll open an issue for it as it's not related to this PR.

Reply:
Ref https://github.com/inspektor-gadget/inspektor-gadget/issues/2000